### PR TITLE
Update the readme files to be correct for the upcoming LTS snapshot z…

### PR DESCRIPTION
**Pull request by RichardBarry**
Description
-----------
Update readme files ready for imminent LTS snapshot release.

Test Steps
-----------
N/A no code changes.

Related Issue
-----------
N/A no code changes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
 called the Windows simulator) to enable their evaluation using the free Visual
 Studio tools and without needing specific microcontroller hardware.
-
-Additional details can be found at: 
 https://www.freertos.org/FreeRTOS-Windows-Simulator-Emulator-for-Visual-Studio-and-Eclipse-MingW.html
+provides additional information on using the Windows port.
 
-Instructions for configuring and using the FreeRTOS IoT libraries are in the
-following links:
+The following links provide instructions on using using the FreeRTOS IoT
```
Comment (aggarw13) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt:24]: "...using ~~using~~ the FreeRTOS..."

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/readme.txt [hunk 2]
```diff
@@ -1,36 +1,53 @@
...
-We are currently working to improve modularity and memory usage of FreeRTOS 
-libraries to ensure the FreeRTOS LTS release helps our users cover a wider set of use cases.
-
-For additional details on LTS and the supported libraries,
-refer to https://freertos.org/ltsroadmap.html
+We are currently working to improve the modularity and memory usage of new
+FreeRTOS IoT libraries before releasing the libraries into the main FreeRTOS
+download with long term support (LTS).  https://freertos.org/ltsroadmap.html
+contains a description of the libraries target modularity and code quality
+criteria.  An LTS release is only updated with bug and vulnerability patches,
```
Comment (aggarw13) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/readme.txt:7]: Seems like the sentence is missing a preposition?
```suggestion
download with long term support (LTS).  https://freertos.org/ltsroadmap.html
contains a description of the libraries that meet the modularity and code quality
criteria.  An LTS release is only updated with bug and vulnerability patches,
```

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt [hunk 3]
```diff
@@ -35,6 +37,11 @@ following links:
 The Visual Studio projects for each of the FreeRTOS IoT library examples are
 located in sub-directories of the following top-level directories:
 
+  Demos using the refactored MQTT library:
+  + /FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_plain_text
```
Comment (aggarw13) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt:41]: Incorrect path. Need to point to the Demo directory!

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt [hunk 4]
```diff
@@ -35,6 +37,11 @@ following links:
 The Visual Studio projects for each of the FreeRTOS IoT library examples are
 located in sub-directories of the following top-level directories:
 
+  Demos using the refactored MQTT library:
+  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_plain_text
+  + /FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_light_weight
```
Comment (muneebahmed10) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt:42]:
```suggestion
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_light_weight
```

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/readme.txt [hunk 5]
```diff
@@ -1,36 +1,53 @@
...
 
 *** LOCATING THE EXAMPLE PROJECTS ***
 
 The Visual Studio projects for each of the FreeRTOS IoT library examples are
 located in sub-directories of the following top-level directories:
 
-  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt
+  Demos using the refactored MQTT library:
+  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_plain_text
+  + /FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_light_weight
```
Comment (muneebahmed10) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/readme.txt:42]:
```suggestion
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_light_weight
```

File: FreeRTOS-Plus/Demo/readme.txt [hunk 6]
```diff
@@ -35,6 +37,11 @@ following links:
 The Visual Studio projects for each of the FreeRTOS IoT library examples are
 located in sub-directories of the following top-level directories:
 
+  Demos using the refactored MQTT library:
+  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_plain_text
+  + /FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_light_weight
```
Comment (muneebahmed10) [inline FreeRTOS-Plus/Demo/readme.txt:42]:
```suggestion
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_light_weight
```

File: README_FIRST.txt [hunk 7]
```diff
@@ -35,6 +37,11 @@ following links:
 The Visual Studio projects for each of the FreeRTOS IoT library examples are
 located in sub-directories of the following top-level directories:
 
+  Demos using the refactored MQTT library:
+  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_plain_text
+  + /FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_light_weight
```
Comment (muneebahmed10) [inline README_FIRST.txt:42]:
```suggestion
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt_light_weight
```

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt (hunk 4 above)
Comment (muneebahmed10) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta1/readme.txt:42]:
```suggestion
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_plain_text
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_light_weight
```

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/readme.txt (hunk 5 above)
Comment (muneebahmed10) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/readme.txt:42]:
```suggestion
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_plain_text
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_light_weight
```

File: FreeRTOS-Plus/Demo/readme.txt (hunk 6 above)
Comment (muneebahmed10) [inline FreeRTOS-Plus/Demo/readme.txt:42]:
```suggestion
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_plain_text
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_light_weight
```

File: README_FIRST.txt (hunk 7 above)
Comment (muneebahmed10) [inline README_FIRST.txt:42]:
```suggestion
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_plain_text
  + /FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_light_weight
```

**Pull request merged**